src/index.js

```diff
@@ -35,6 +35,10 @@ Builder.prototype.build = function (path, options) {
   return this.$build(path, toHash(options));
 };
 
+Builder.prototype.buildStr = function (str, path = ".", options) {
```

Comment:
I think we should call it `buildString`. I believe that the `str` abbreviation is not really popular in the JavaScript world and `buildString` is also consistent with `toString`.